### PR TITLE
[res] Add examples for NonMaxSuppressionV5 op

### PR DESCRIPTION
**Pull request by ai-moiseev**
Add examples for NonMaxSuppressionV5 operator.

ONE-DCO-1.0-Signed-off-by: Alexander Moiseev <a.moiseev@samsung.com>

### CODE REVIEW

File: res/TensorFlowPythonExamples/examples/non_max_suppression_with_scores/__init__.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import tensorflow as tf
+
+max_output_size = tf.compat.v1.constant(4)
+
+in_boxes_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(8, 4), name="Hole")
+in_scores_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(8), name="Hole")
+non_max_suppression_with_scores_ = tf.compat.v1.image.non_max_suppression_with_scores(
```
Comment (seanshpark) [inline res/TensorFlowPythonExamples/examples/non_max_suppression_with_scores/__init__.py:7]: Error with TF1.13.x
> AttributeError: module 'tensorflow._api.v1.compat.v1.image' has no attribute 'non_max_suppression_with_scores'

If we should run in higher versions, please add a note about it.

File: res/TensorFlowPythonExamples/examples/non_max_suppression_with_scores_2/__init__.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+max_output_size = tf.compat.v1.constant(6)
+iou_threshold = tf.compat.v1.constant(0.5)
+score_threshold = tf.compat.v1.constant(0.6)
+soft_nms_sigma = tf.compat.v1.constant(0.5)
+
+in_boxes_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(12, 4), name="Hole")
+in_scores_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(12), name="Hole")
+
+non_max_suppression_with_scores_ = tf.compat.v1.image.non_max_suppression_with_scores(
```
Comment (seanshpark) [inline res/TensorFlowPythonExamples/examples/non_max_suppression_with_scores_2/__init__.py:11]: > AttributeError: module 'tensorflow._api.v1.compat.v1.image' has no attribute 'non_max_suppression_with_scores'

ditto

File: res/TensorFlowPythonExamples/examples/non_max_suppression_with_scores/__init__.py (hunk 1 above)
Comment (ai-moiseev) [reply res/TensorFlowPythonExamples/examples/non_max_suppression_with_scores/__init__.py:9]: It seems ``non_max_suppression_with_scores`` was introduced in TF 1.15, I've added comment about version requirement.

File: res/TensorFlowPythonExamples/examples/non_max_suppression_with_scores_2/__init__.py (hunk 2 above)
Comment (ai-moiseev) [reply res/TensorFlowPythonExamples/examples/non_max_suppression_with_scores_2/__init__.py:12]: Done, added line comment about TF version requirement for `non_max_suppression_with_scores`.

**Pull request merged**